<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>400</width>
      <height>510</height>
     </rect>
    </property>
    <property name="title">
     <string>Eulerain Magnified Video</string>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>470</y>
       <width>150</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>360</width>
       <height>381</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>470</y>
       <width>150</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Play EVM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>429</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>429</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>429</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>440</width>
      <height>510</height>
     </rect>
    </property>
    <property name="title">
     <string>FLIR Interpolated Video</string>
    </property>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>239</x>
       <y>470</y>
       <width>150</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>430</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>430</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>29</y>
       <width>360</width>
       <height>371</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>430</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>470</y>
       <width>150</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Play FLIR</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>440</width>
      <height>510</height>
     </rect>
    </property>
    <property name="title">
     <string>Overlapped Video</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>430</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>470</y>
       <width>150</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Overlap Videos</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>430</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>430</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>57</x>
       <y>470</y>
       <width>150</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Play Processed Video</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>360</width>
       <height>371</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
